### DISCUSSION TITLE
Preterm Infant simulation not working

### DISCUSSION THREAD

**Discussion by mmalsmadi** [Open-Systems-Pharmacology/Forum#758] [Questions & Problems] [answered]
Hello,

I am working with a project where I am scaling an adult PBPK model to a premature infant population (GA=24 weeks)
When I ran the adult simulation, everything went fine and got a decent simulation
But, when I changed the population to premature infant described above, the simulation resulted in no lines of model-predicted levels, even that PK-Sim did NOT indicate that the simulation failed to integrate and the simulation went to the end

Any suggestions on why did this happen?
In my model, I have three enzymes 3A4, 2D6, and 3A5 in addition to renal glomerular filtration

I tried two settings;
1- put the values of the expression levels in preterm infant as 100% of the adult values
2- Put the expression levels of CYP3A4, CYP3A5, CYP2D6 in the preterm infant as 3%, 3%, and 5%, respectively from the adult reference value  

Both settings resulted in the same problem described above
Does PK-Sim scale the expression levels automatically or not (i.e. should I do the setting 1, or 2)?

Thank you for the help

**Comment by prvmalik** [comment]
Hi, the preterm population is not validated for oral simulations and will not work. Are you performing an oral simulation?

**Comment by mmalsmadi** [comment]
yes, I did an oral simulation
Is there a way to go around this problem?

**Comment by prvmalik** [comment]
No easy workaround. A parameterization would have to be developed and evaluated for the preterm infant gut. One has been proposed here: Abduljalil et al., 2020, doi: 10.1007/s40262-019-00825-6

**Comment by mmalsmadi** [comment]
many thanks

**Comment by mmalsmadi** [comment]
will this need dealing with MoBi or purely done via PK-Sim?

**Comment by mmalsmadi** [comment]
Also, assume if I want to do a preterm infant simulation after intravenous dosing
for setting the enzyme expression levels, which approach should I use?

1- put the values of the expression levels in preterm infant as 100% of the adult values
2- Put the expression levels of CYP3A4, CYP3A5, CYP2D6 in the preterm infant as 3%, 3%, and 5%, respectively from the adult reference value


i.e. does PK-Sim scale the expression levels automatically or not?

**Comment by prvmalik** [comment, marked as answer]
Yes PKSim will scale the ontogeny of these three enzymes automatically (therefore do #1)

To accomplish this, the 'ontogeny/variability like' option must be designated for the appropriate CYP enzyme, as shown here.
![Pic2](https://user-images.githubusercontent.com/29066045/82704132-eca17400-9c42-11ea-87b9-2f911a2c4ac2.JPG)

Alternatively you can do the scaling manually (#2) but you must ensure that there is no link to the GeneDB and that the 'ontogeny/variability like' option is designated as 'Undefined.'

**Comment by mmalsmadi** [comment]
thank you for the explanation above,

going back to the oral stimulation in Preterm Infant;
will this need dealing with MoBi or purely done via PK-Sim to do the scaling of GI anatomy and physiology?

**Comment by prvmalik** [comment]
I have not done this before. I think maybe others in the community have done it privately. I will let them respond as to the best course of action.
